docs/orleans/streaming/broadcast-channel.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Broadcast channels
+description: Learn how to work with Orleans broadcast channels.
+ms.date: 12/14/2022
+---
+
+# Broadcast channels in Orleans
+
+Broadcast channels are a special type of streaming mechanism that can be used to broadcast messages to all connected clients. Broadcast channels are useful for scenarios where you want to send a message within a producer and consumer model, but you don't want to keep track of the consumers. For example, you can use broadcast channels to send a message to all connected clients when a new user joins a chat room.
```

Comment:
I think we may want to avoid the word "streaming" where possible. 

"Broadcast channels are a special type of broadcasting mechanism that can be used to send messages to all subscribers".

I also think it would be useful to say that it is using regular grain calls for that, unlike the decoupled producer and consumer model from the streaming infrastructure